src/test/java/rs/hybridit/challenges/ChallengesTest.java

```diff
@@ -83,7 +85,7 @@ public void findMissingNumberInArrayTest() {
         //test2
         //given
```

Comment:
Remove code that is commented out, comments should only contain relevant information, not code